.azure-pipelines/packages.yml

```diff
@@ -106,6 +106,25 @@ jobs:
     vmImage: ubuntu-16.04
 
   steps:
+  - task: DotNetCoreInstaller@0
+    displayName: install dotnet core sdk
+    inputs:
+      version: $(dotnetCoreSdkVersion)
```

Comment:
Using the CI steps to build the .NET core library can help make sure we use the same .NET Core SDK across our packaging build.